packages/backend/src/open_payments/quote/model.ts

```diff
@@ -152,6 +153,10 @@ export class Quote extends WalletAddressSubresource {
     this.highEstimatedExchangeRateDenominator = value.b.value
   }
 
+  public get method(): 'ilp' {
+    return 'ilp'
+  }
+
```

Comment:
@mkurapov wasn't quite sure what our intention is for modeling payment methods.

The new openapi spec requires the quote type in `POST /quote` body but I didn't actually add it to the create path (`quoteRoutes.create`, `quoteService.create`, `createQuote` test util etc) since its not stored anywhere and I'm not sure how we want to do that (method column on Quote? fkey column on Quote point to Methods table?)

Reply:
I like having the idea of a `PaymentMethods` table that determines what types of payment methods are active or inactive, but for the sake of time this (hard-coding ilp) works well